pom.xml

```diff
@@ -187,5 +188,37 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>xml-check-format</id>
+            <modules>
+                <module>terasoluna-gfw-common-libraries</module>
+                <module>terasoluna-gfw-dependencies</module>
+            </modules>
+            <build>
+                <pluginManagement>
+                    <plugins>
+                        <plugin>
+                            <groupId>org.codehaus.mojo</groupId>
+                            <artifactId>xml-maven-plugin</artifactId>
+                            <version>${xml-maven-plugin.version}</version>
+                            <configuration>
+                                <indentSize>2</indentSize>
+                                <formatFileSets>
+                                    <formatFileSet>
+                                        <directory>${project.basedir}</directory>
```

Comment:
Sorry . I forgot to reflect the contents of the 2a196186373104334f4882c6e3c4ac5798f0fc05 commit .
I cherry-picked above commit at 6286158d43d78d7c80cb869df11719f295622965 commit .